inputs/demand/industry/industry_useful_demand_for_chemical_fertilizers_electricity_efficiency.ad

```diff
@@ -1,10 +1,10 @@
 - query = UPDATE(V(industry_useful_demand_for_chemical_fertilizers_electricity), preset_demand, NEG(USER_INPUT()))
 - priority = 0
-- max_value = 5.0
-- min_value = 0.0
+- max_value = 100.0
+- min_value = 0
```

Comment:
For consistency, I would denote these inputs with 1 decimal: `- min_value = 0.0`